clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/VeniceChangelogConsumer.java

```diff
@@ -58,6 +58,35 @@
    */
   CompletableFuture<Void> seekToEndOfPush(Set<Integer> partitions);
 
+  /**
+   * Pause the client on all subscriptions. See {@link #pause(Set)} for more information.
+   *
+   * @throws VeniceException if seek operation failed for any of the partitions.
```

Comment:
Minor:    @throws `VeniceException` if `pause` operation failed for any of the partitions, serveral cases as below.